Modules/Core/Common/include/itkMacro.h

```diff
@@ -151,19 +151,15 @@ namespace itk
 #endif
 
 /*
- * ITK only supports MSVC++ 14.0 and greater
- * MSVC++ 14.0 _MSC_VER == 1900 (Visual Studio 2015 version 14.0)
- * MSVC++ 14.1 _MSC_VER == 1910 (Visual Studio 2017 version 15.0)
- * MSVC++ 14.11 _MSC_VER == 1911 (Visual Studio 2017 version 15.3)
- * MSVC++ 14.12 _MSC_VER == 1912 (Visual Studio 2017 version 15.5)
+ * ITK only supports MSVC++ 14.14 and greater
  * MSVC++ 14.13 _MSC_VER == 1913 (Visual Studio 2017 version 15.6)
```

Comment:
Should this line be removed too?

Reply:
Thanks again, addressed with the latest [force-push](https://github.com/InsightSoftwareConsortium/ITK/compare/57fd767dc14f8b1d1d6d9ecf44cf48454707e649..de7124eacb3466359c1ba529f2e83b4d183ba74f)